src/Utilities/Idm/SourceLoad.F90

```diff
@@ -168,6 +171,13 @@ subroutine load_simnam()
       !
       ! -- open namfile and load to input context
       call input_load(simfile, mf6_input, simfile, iout)
+      !
+      ! -- load optional HPC configuration file
+      if (filein_fname(hpc6_filename, 'HPC6_FILENAME', mf6_input%mempath, &
+                       simfile)) then
+        hpc_input = getModflowInput('HPC6', 'UTL', 'HPC', 'UTL', 'HPC', simfile)
```

Comment:
Line 178 can remove the optional argument "simfile" as it is used to distinguish an array based package input set.